workshops/splatter_paint/es-xl.md

```diff
@@ -0,0 +1,191 @@
+---
+name: 'Splatter Paint'
+description: 'Crazy colorful splatter paint in your browser with Paper.js'
```

Comment:
Don't forget to translate the title/description as well.

Reply:
Sure! It's done now, thanks @eilla1 .